mreg_cli/api/models.py

```diff
@@ -256,6 +256,41 @@ def valid_numeric_ttl(self, ttl: int) -> int:
         return ttl
 
 
+class WithName:
+    """Model for an object that has a name element."""
+
```

Comment:
We can, theoretically, add the name field in here? Although, we may have to handle types if the name field is either a str or a HostT... :(